esp/platform/espp.cpp

```diff
@@ -432,6 +433,44 @@ static IPropertyTree * extractLegacyOptions(IPropertyTree * legacyOptions)
     return extractedOptions.getClear();
 }
 
+bool startEspEventRecording(const char * options, const char * filename)
+{
+    if (isEmptyString(options))
+        options = "threadid";
```

Comment:
This does match roxie, and may be a reasonable default. Is it too early to check if trace is enabled and, if enabled, default to include the trace ID?

Reply:
No, that would be a sensible move.